yarn-project/cli-wallet/src/cmds/cancel_tx.ts

```diff
@@ -20,13 +21,20 @@ export async function cancelTx(
     return;
   }
 
+  const maxFeesPerGas = new GasFees(
```

Comment:
Do we need to also increase the max fee per gas? I'd expect only increasing priority fees for cancellation :thinking:

Reply:
I included max fees because the gasSettings passed to this function is fetched from the previous tx. So the max fees might be tailored to only fit the previous tx's fees. But now you pointed this out, I guess a better approach is for the user to also be able to increase the max fees for this new tx. Will change it now! Thanks 😃